components/edge-identity/backend/apps/default/db.py

```diff
@@ -0,0 +1,52 @@
+import sqlite3
+from typing import Any, List, Union
+
+import click
+from flask import current_app, g
+from flask.cli import with_appcontext
+
+DATABASE = "edge-identity.db"
+
+
+def get_db():
+    if "db" not in g:
+        g.db = sqlite3.connect(DATABASE, detect_types=sqlite3.PARSE_DECLTYPES)
```

Comment:
🤔 Not relevant for this PR, but I'm wondering what it takes to change the storage backend... thinking that we might be able to utilize `etcd` and have this service just create the current resource plus an operator which in turn configures spire and so on...

Reply:
It's an approach worth checking. Let's discuss this offline and comeup with follow-ups.